readthedocs/projects/models.py

```diff
@@ -1109,3 +1130,19 @@ def get_feature_display(self):
         implement this behavior.
         """
         return dict(self.FEATURES).get(self.feature_id, self.feature_id)
+
+
+class EnvironmentVariable(TimeStampedModel, models.Model):
+    name = models.CharField(
+        max_length=128,
+        help_text=_('Name of the environment variable'),
+    )
+    value = models.CharField(
+        max_length=256,
+        help_text=_('Value of the environment variable'),
+    )
+    project = models.ForeignKey(
+        Project,
+        on_delete=models.CASCADE,
+        help_text=_('Project where this variable will be used'),
```

Comment:
Any particular thoughts on using a reverse name? Historically, we've given a verbose reverse name like "environment_variables" or something other than the default "environmentvariable_set"

Reply:
In particular in this case, I'm using `environment_variables` as a property that returns the dictionary like `{"VARIABLE": "value"}` and `environmentvariable_set.` to make queries on this relation.

Reply:
This seems fair. I don't have any particular opinions or suggestions here, so probably fine to continue with a pattern like this. Perhaps there is a more Django-y pattern for verbose property name/verbose field name, instead of verbose property name/obtuse field name, but :man_shrugging: